app/src/main/java/com/example/unlibrary/unlibrary/UnlibraryViewModel.java

```diff
@@ -200,6 +205,15 @@ public void handoff(String isbn) {
         }
     }
 
+    /**
+     * Getter for the mCurrentBookOwner object.
+     *
+     * @return LiveData<User> This returns the mCurrentBookOwner object
+     */
+    public LiveData<User> getCurrentBookOwner() {
```

Comment:
```suggestion
    public MutableLiveData<User> getCurrentBookOwner() {
```
Honestly curious what would've happened if we kept it as LiveData